### PR TITLE
Remove twitter-blocked logic

### PR DESCRIPTION
**Pull request by Phaeton**
## Describe your changes
Move twitter block hex to main db, remove twitter files and checks.

## Checklist before requesting a review

- [X] I have performed a self-review of my code.
- [X] While waiting for someone to review your request, please help review [another open pull request](https://github.com/sdr-enthusiasts/plane-alert-db/pulls) to support the maintainers.

### CODE REVIEW

File: plane-alert-db.csv [hunk 1]
```diff
@@ -13296,4 +13296,15 @@ AD18D5,N943FL,Ron Desantis,Cessna Citation Latitude,C68A,Civ,Bizjet,Fly Navy,Flo
 AB772B,N838MF,Ronald Perelman,Gulfstream G650,GLF6,Civ,Bizjet,Revlon,Political Donor,Don't you know who I am?,https://en.wikipedia.org/wiki/Ronald_Perelman
 A97659,N709DS,Steve Ballmer,Gulfstream G650,GLF6,Civ,Bizjet,Microsoft,LA Clippers,Don't you know who I am?,https://en.wikipedia.org/wiki/Steve_Ballmer
 AC701E,N900KS,Steven Spielberg,Gulfstream G650,GLF6,Civ,Bizjet,Jurassic Park,Indiana Jones,Don't you know who I am?,https://en.wikipedia.org/wiki/Steven_Spielberg
-AD5B90,N96UA,Under Armour Corporation,Gulfstream G-V,GLF5,Civ,Bizjet,Undies,As Seen on TV,Vanity Plate,https://en.wikipedia.org/wiki/Under_Armour
\ No newline at end of file
+AD5B90,N96UA,Under Armour Corporation,Gulfstream G-V,GLF5,Civ,Bizjet,Undies,As Seen on TV,Vanity Plate,https://en.wikipedia.org/wiki/Under_Armour
+A2AE0A,N272BG,Falcon Landing LLC,Gulfstream G550,GLF5,Civ,Elon Musk,SpaceX,Toys4Billionaires,Don't you know who I am ?,https://youtu.be/bvim4rsNHkQ
```
Comment (dziban303) [inline plane-alert-db.csv:13300]:
```suggestion
A2AE0A,N272BG,Falcon Landing LLC,Gulfstream G550,GLF5,Civ,Elon Musk,SpaceX,Toys4Billionaires,Don't you know who I am?,https://youtu.be/bvim4rsNHkQ

```

Comment (dziban303) [reply plane-alert-db.csv:13300]: Actually needs to be on all these, removing the space: `am ?`

File: plane-alert-db.csv [hunk 2]
```diff
@@ -13296,4 +13296,15 @@ AD18D5,N943FL,Ron Desantis,Cessna Citation Latitude,C68A,Civ,Bizjet,Fly Navy,Flo
...
+AD5B90,N96UA,Under Armour Corporation,Gulfstream G-V,GLF5,Civ,Bizjet,Undies,As Seen on TV,Vanity Plate,https://en.wikipedia.org/wiki/Under_Armour
+A2AE0A,N272BG,Falcon Landing LLC,Gulfstream G550,GLF5,Civ,Elon Musk,SpaceX,Toys4Billionaires,Don't you know who I am ?,https://youtu.be/bvim4rsNHkQ
+A64304,N502SX,Falcon Landing LLC,Gulfstream G550,GLF5,Civ,Elon Musk,SpaceX,Toys4Billionaires,Don't you know who I am ?,https://youtu.be/bvim4rsNHkQ
+A835AF,N629TS,Falcon Landing LLC,Gulfstream G650,GLF6,Civ,Elon Musk,SpaceX,Toys4Billionaires,Don't you know who I am ?,https://youtu.be/bvim4rsNHkQ
+A34877,N310SX,Falcon Landing LLC,Not Yet Allocated,ZZZZ,Civ,Elon Musk,SpaceX,Toys4Billionaires,Don't you know who I am ?,https://youtu.be/bvim4rsNHkQ
+AB1270,N812SX,Falcon Landing LLC,Not Yet Allocated,ZZZZ,Civ,Elon Musk,SpaceX,Toys4Billionaires,Don't you know who I am ?,https://youtu.be/bvim4rsNHkQ
+AB817F,N840SX,Falcon Landing LLC,Not Yet Allocated,ZZZZ,Civ,Elon Musk,SpaceX,Toys4Billionaires,Don't you know who I am ?,https://youtu.be/bvim4rsNHkQ
+ACDD88,N928SX,Falcon Landing LLC,Not Yet Allocated,ZZZZ,Civ,Elon Musk,SpaceX,Toys4Billionaires,Don't you know who I am ?,https://youtu.be/bvim4rsNHkQ
```
Comment (dziban303) [inline plane-alert-db.csv:13306]:
```suggestion
A64304,N502SX,Falcon Landing LLC,Gulfstream G550,GLF5,Civ,Elon Musk,SpaceX,Toys4Billionaires,Don't you know who I am?,https://youtu.be/bvim4rsNHkQ
A835AF,N629TS,Falcon Landing LLC,Gulfstream G650,GLF6,Civ,Elon Musk,SpaceX,Toys4Billionaires,Don't you know who I am?,https://youtu.be/bvim4rsNHkQ
A34877,N310SX,Falcon Landing LLC,Not Yet Allocated,ZZZZ,Civ,Elon Musk,SpaceX,Toys4Billionaires,Don't you know who I am?,https://youtu.be/bvim4rsNHkQ
AB1270,N812SX,Falcon Landing LLC,Not Yet Allocated,ZZZZ,Civ,Elon Musk,SpaceX,Toys4Billionaires,Don't you know who I am?,https://youtu.be/bvim4rsNHkQ
AB817F,N840SX,Falcon Landing LLC,Not Yet Allocated,ZZZZ,Civ,Elon Musk,SpaceX,Toys4Billionaires,Don't you know who I am?,https://youtu.be/bvim4rsNHkQ
ACDD88,N928SX,Falcon Landing LLC,Not Yet Allocated,ZZZZ,Civ,Elon Musk,SpaceX,Toys4Billionaires,Don't you know who I am?,https://youtu.be/bvim4rsNHkQ

```

**Review by dziban303 — APPROVED** [review]

File: plane-alert-db.csv [hunk 3]
```diff
@@ -13296,4 +13296,15 @@ AD18D5,N943FL,Ron Desantis,Cessna Citation Latitude,C68A,Civ,Bizjet,Fly Navy,Flo
...
+AB1270,N812SX,Falcon Landing LLC,Not Yet Allocated,ZZZZ,Civ,Elon Musk,SpaceX,Toys4Billionaires,Don't you know who I am?,https://youtu.be/bvim4rsNHkQ
+AB817F,N840SX,Falcon Landing LLC,Not Yet Allocated,ZZZZ,Civ,Elon Musk,SpaceX,Toys4Billionaires,Don't you know who I am?,https://youtu.be/bvim4rsNHkQ
+ACDD88,N928SX,Falcon Landing LLC,Not Yet Allocated,ZZZZ,Civ,Elon Musk,SpaceX,Toys4Billionaires,Don't you know who I am?,https://youtu.be/bvim4rsNHkQ
+A0FE3B,N16DJ,Elon Musk,Gulfstream G-IV,GLF4,Civ,PIA,Hive Mind,You Can Run But You Cannot Hide,PIA,https://en.wikipedia.org/wiki/Elon_Musk
+A0FEBB,N628TS,Elon Musk,Gulfstream G650,GLF6,Civ,PIA,Hive Mind,You Can Run But You Cannot Hide,PIA,https://en.wikipedia.org/wiki/Elon_Musk
+AA3410,N757AF,DJT Operations,Boeing 757,B752,Civ,Original Nuttah,Orange Danger,Lock Him Up,Dictator Alert,https://en.wikipedia.org/wiki/Veracity_of_statements_by_Donald_Trump
+A9B67C,N725DT,Donald Trump,Cessna Citation X,C750,Civ,Bad Human,Orange Danger,Lock Him Up,Dictator Alert,https://en.wikipedia.org/wiki/Veracity_of_statements_by_Donald_Trump
```
Comment (pmd5700) [inline plane-alert-db.csv:13310]:
```suggestion
A9B67C,N725DT,Donald Trump,Cessna Citation X,C750,Civ,Bad Human,Orange Danger,Lock Him Up,Dictator Alert,https://en.wikipedia.org/wiki/Veracity_of_statements_by_Donald_Trump
AE1E43,07-3867,USAF,Raytheon T-6A Texan II,TEX2,Mil,Training Wheels,Bicycle Made For Two,To Fly Fight And Win,USAF,https://www.airforce.com
AAB8CD,N790J,IBM Corp,Dassault Falcon 2000,F2TH,Civ,HAL,Counting Machines,Big Blue,As Seen on TV,https://en.wikipedia.org/wiki/IBM
AE059D,60-0362,USAF,Boeing KC-135R,K35R,Mil,Air2Air,I Am Old,To Fly Fight And Win,USAF,https://www.airforce.com
C059CB,C-GIAC,Nutrien,Gulfstream G-V,GLF5,Civ,Potash,Phosphate,Fertilizer,As Seen on TV,https://www.nutrien.com/
AE0990,91-26327,United States Army,UH-60L Blackhawk,H60,Mil,Get To The Choppa,Military Transport,Blackhawk,Toy Soldiers,https://en.wikipedia.org/wiki/Sikorsky_UH-60_Black_Hawk
AA688A,N77VA,Tom Cruise,Honda Jet,HDJT,Civ,Maverick,Ethan Hunt,Xenu,Don't you know who I am?,https://youtu.be/EPzvExtJgxw
AE0C60,93-26485,United States Army,UH-60L Blackhawk,H60,Mil,Get To The Choppa,Military Transport,Blackhawk,Toy Soldiers,https://en.wikipedia.org/wiki/Sikorsky_UH-60_Black_Hawk
```

Comment (pmd5700) [reply plane-alert-db.csv:13310]: Not sure if this the proper way to add while there's an open Pull Request.

Comment (pmd5700) [reply plane-alert-db.csv:13310]:
```
AE1E43,https://live.staticflickr.com/65535/32449082503_a709e879ad_k.jpg,,,
AAB8CD,https://cdn.jetphotos.com/full/5/53290_1564999840.jpg,https://cdn.jetphotos.com/full/5/65924_1539690442.jpg,https://cdn.jetphotos.com/full/5/31764_1525964616.jpg,https://cdn.jetphotos.com/full/6/42615_1524307831.jpg
AE059D,https://www.airport-data.com/images/aircraft/001/237/001237375.jpg,https://www.airport-data.com/images/aircraft/001/396/001396283.jpg,https://live.staticflickr.com/7845/47103474781_85c918a60d_b.jpg,,
C059CB,https://cdn.jetphotos.com/full/5/44381_1669989765.jpg,https://cdn.jetphotos.com/full/6/18342_1641152952.jpg,https://cdn.jetphotos.com/full/6/56620_1634432955.jpg,https://cdn.jetphotos.com/full/6/40080_1584464325.jpg
AE0990,https://www.airhistory.net/photos/0152989.jpg,,,
AA688A,https://cdn.jetphotos.com/full/6/79368_1568399736.jpg,https://live.staticflickr.com/65535/50248343478_50a6fad829_4k.jpg,https://live.staticflickr.com/65535/50248985646_2ec0930e0a_h.jpg,https://live.staticflickr.com/65535/50249187042_6708d2c1a4_k.jpg
AE0C60,,,,
```

**Pull request merged**